server/database/connection.js

```diff
@@ -1,12 +1,14 @@
 const { Pool } = require('pg');
```

Comment:
change your `require` to `import`

Reply:
I fixed this,
Converted connection.js file to ES6